src/styles/_tabs.scss

```diff
@@ -0,0 +1,57 @@
+@import 'variables';
+
+.react-tabs {
+  -webkit-tap-highlight-color: transparent;
+}
+
+.react-tabs__tab-list {
+  margin: 0 0 10px;
+  padding: 30px 0 0 30px;
+  background-color: $gray-lighter;
+}
+
+.react-tabs__tab {
+  display: inline-block;
+  border-bottom: none;
+  bottom: -1px;
+  position: relative;
+  list-style: none;
+  padding: 6px 12px;
+  cursor: pointer;
+}
+
+.react-tabs__tab--selected {
+  background: #fff;
+  border-color: #aaa;
```

Comment:
I didn't see this color specifically in the variables file.  Is there another one we should be using or should this be added to the palette in `_variables.scss`?